### PR TITLE
v0.8.0

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
**v0.8.0** January 20, 2022

* Enhancements
    * Add issue templates for bugs, feature requests and documentation improvements (#271)
* Changes
    * Update pip to 21.3.1 for test requirements (#265)
    * Restrict to Python 3.7 to 3.9 (#265)
    * Use black and remove autopep8 for linting (#265)
    * Update minimum dependency checker with the correct reviewers (#267)
    * Rename ``LabelMaker.target_entity`` to ``LabelMaker.target_dataframe_name`` (#276)
* Documentation Changes
    * Update install instructions to specify correct python versions (#265)
    * Update example notebooks to use latest Featuretools and EvalML APIs (#275)
* Testing Changes
    * Add unit test for dropping empty data slices (#280)
    * Add auto approve workflow for dependency updates (#281)

### CODE REVIEW

**Review by gsheni — APPROVED** [review]

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -20,8 +28,8 @@ Future Release
         * Add unit test for dropping empty data slices (:pr:`280`)
         * Add auto approve workflow for dependency updates (:pr:`281`)
 
-    | Thanks to the following people for contributing to this release:
-    | :user:`gsheni`, :user:`jeff-hernandez`, :user:`thehomebrewnerd`
+    Thanks to the following people for contributing to this release:
+    :user:`gsheni`, :user:`jeff-hernandez`, :user:`thehomebrewnerd`
```
Comment (jeff-hernandez) [inline docs/source/release_notes.rst:32]: Can we keep on separate lines?

Screen Shot 2022-01-20 at 12 27 21 PM

Comment (jeff-hernandez) [reply docs/source/release_notes.rst]: Or we can also format the other entries -- main thing is just using a consistent format

Comment (thehomebrewnerd) [reply docs/source/release_notes.rst]: Yeah, I changed to match what we have been doing in other repos, but I didn't catch the other entries were like this too. I'll change this back for now to use separate lines to be consistent.

**Review by jeff-hernandez — APPROVED** [review]
